docs/UserCommands.md

```diff
@@ -0,0 +1,44 @@
+## User Command Reference
+
+
+`@rocket user`
+
+## Options to specify input:
+
+* `-edit`
+    * `-name` somename `-email` email `-pos` yourposition `-github` githublink `-major` yourmajor `-bio` yourbio
```

Comment:
A small usage thing: with Unix commands there's usually a "long" form that begins with two dashes, and sometimes a "short" form that's one dash followed by one letter. I would prefer that here, so that you could do for instance either `--name` or `-n` (and similarly for all the other options).